### PR TITLE
Add PgpKey.Emails and tests

### PR DESCRIPTION
**Pull request by paulfurley**
Add `exampledata` package with a useful example key

### CODE REVIEW

**Comment by paulfurley** [comment]
Turns out `testdata` was a bad naming choice and confused the compiler. Closing for now.

File: pgpkey/pgpkey_test.go [hunk 1]
```diff
@@ -52,6 +53,49 @@ func TestEmailMethod(t *testing.T) {
...
+
+		if len(got) != len(expected) {
+			t.Fatalf("Expected %d emails, got %d: %v", len(expected), len(got), got)
+		}
+
+		for i := range expected {
+			if expected[i] != got[i] {
+				t.Fatalf("expected[%d] = '%s', got = '%s'", i, expected[i], got[i])
+			}
+		}
```
[the remark this answers is not in the document]
Comment (paulfurley) [reply pgpkey/pgpkey_test.go]: Definitely, and use it in the other 15 places we've implemented that. I propose we (later) yet-another-package implementing some assert helpers

File: exampledata/exampledata.go [hunk 2]
```diff
@@ -0,0 +1,90 @@
+package exampledata
+
+import (
+	"github.com/fluidkeys/fluidkeys/fingerprint"
+)
+
+const ExamplePublicKey3 = `
```
Comment (idrysdale) [inline exampledata/exampledata.go]: Why 3? 3 Uids?

Comment (idrysdale) [reply exampledata/exampledata.go]: Ok!

**Pull request merged**